cmd/schemagen/schemagen.go

```diff
@@ -82,6 +85,13 @@ func renderTemplate(md *gocql.KeyspaceMetadata) ([]byte, error) {
 		log.Fatalln("unable to parse models template:", err)
 	}
 
+	for _, name := range ignoredSchemaElements(md) {
+		delete(md.Tables, name)
+	}
```

Comment:
```suggestion
	ignoredNames := make(map[string]struct{})
	for _, ignoredName := range strings.Split("", ",") {
		ignoredNames[ignoredName] = struct{}{}
	}
	
	if ignoreIndexes {
		for indexName := range md.Indexes {
			ignoredNames[indexName] = struct{}{}
		}
	}
	
	maps.DeleteFunc(md.Tables, func(k string, v *gocql.TableMetadata) bool {
		_, ok := ignoredNames[k]
		return ok
	})
```

Reply:
```suggestion
	ignoredNames := make(map[string]struct{})
	for _, ignoredName := range strings.Split(*flagIgnoreNames, ",") {
		ignoredNames[ignoredName] = struct{}{}
	}
	if *flagIgnoreIndexes {
		for name := range md.Tables {
			if strings.HasSuffix(name, "_index") {
				ignoredNames[name] = struct{}{}
			}
		}
	}
	for name := range ignoredNames {
		delete(md.Tables, name)
	}
```

The module is set to support go1.17, that makes it impossible to use the `maps` package unfortunately.

```
implicit function instantiation requires go1.18 or later (compiler)
```

Also, there's no `Indexes` field on the `KeyspaceMetadata`.

Reply:
fine, for loop is also fine